gammapy/cube/make.py

```diff
@@ -395,21 +405,59 @@ def make_mask_energy_edisp_bias(self, dataset):
 
         Parameters
         ----------
-        dataset : `SpectrumDataset` or `SpectrumDatasetOnOff`
+        dataset : `MapDataset`, `MapDatasetOnOff`, `SpectrumDataset` or `SpectrumDatasetOnOff`
             Dataset to compute mask for.
 
         Returns
         -------
         mask_safe : `~numpy.ndarray`
             Safe data range mask.
         """
+        edisp = dataset.edisp
+
         if isinstance(dataset, (MapDataset, MapDatasetOnOff)):
-            raise NotImplementedError(
-                "'edisp-bias' method currently only supported for spectral datasets"
-            )
+            if self.position is None:
```

Comment:
Maybe we compute the threshold at the center of the map as a default? However I'm not sure this is a safe assumption. Do you know @registerrier what is the typical quality of IRFs at `offset=0 deg`?